ROMFS/px4fmu_common/init.d/2101_fw_AERT

```diff
@@ -22,4 +22,4 @@ sh /etc/init.d/rc.fw_defaults
 set MIXER AERT
 
 # use PWM parameters for throttle channel
-set PWM_OUT 4
+set PWM_OUT 1234
```

Comment:
This is not necessarily safe (driving servos to min) and we have this in virtually every fixed wing config. The intent is to *not* change the rate but set this one output to PWM_MIN - which is *compatible* with any mask. E.g. see here another (better documented) example:
https://github.com/PX4/Firmware/blob/18ec945b7f76bd84707ce337e08f4fe6bc81aa99/ROMFS/px4fmu_common/init.d/3033_wingwing#L47-L49

What broke it to set this line from "none" to a value indicating to the system to fetch the parameter value:
https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L179

We did that quite a while ago. The proper fix needs cleanup, if this boots right now then I'd say remove the above change and merge the rest.